elixir_grpc/lib/helloworld.pb.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Helloworld.HelloRequest do
```

Comment:
could we generate those in the build phase?

Reply:
I'll give a try